react/TranslateEstimate.js

```diff
@@ -2,50 +2,30 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { injectIntl, intlShape } from 'react-intl'
 
-class TranslateEstimate extends React.Component {
-  constructor(props) {
-    super(props)
-
-    this.state = {
-      id: this.getTranslateId(props.shippingEstimate, props.isPickup),
-      timeAmount: this.getTimeAmount(props.shippingEstimate),
-    }
-  }
-
-  componentDidUpdate() {
-    const { shippingEstimate, isPickup } = this.props
-
-    this.setState({
-      id: this.getTranslateId(shippingEstimate, isPickup),
-      timeAmount: this.getTimeAmount(shippingEstimate),
-    })
-  }
-
-  getTranslateId(shippingEstimate, isPickup) {
-    const shippingEstimateString =
+const getTranslateId = (shippingEstimate, isPickup) => {
+  const shippingEstimateString =
       shippingEstimate && shippingEstimate.split(/[0-9]+/)[1]
```

Comment:
Here you are splitting a date, right? I think that putting the literal 1 inside a const and calling it MONTH would improve readability. Besides, you could use \D to match digits just like you used below

Reply:
It's not a date that is being splitted here. It's a string with this format `1bd`.